codespell_lib/data/dictionary.txt

```diff
@@ -24107,6 +24107,8 @@ linz->lines
 lippizaner->lipizzaner
 liquify->liquefy
 liquour->liquor
+lirarie->librarie
```

Comment:
It's `library`, not `librarie`, isn't it? Actually codespell itself contains a fix for `librarie`:
```
librarie->libraries, library,
```
So perhaps change to:
```
lirarie->library, libraries,
lirary->library
```